lib/model/classycat_classify.py

```diff
@@ -137,17 +140,33 @@ def classify_and_store_results(self, schema_id, items):
 
             result['labels'] = [label for label in result['labels'] if label in permitted_labels]
 
+        # if there is at least one item with labels, save the results to s3
         if not all([len(result['labels']) == 0 for result in final_results]):
             results_file_id = str(uuid.uuid4())
             upload_file_to_s3(self.output_bucket, f"{schema_id}/{results_file_id}.json", json.dumps(final_results))
 
-        return final_results
+            # prepare the final results to be stored in alegre
+            # save "content" and "context"
+            # content is text, doc_id is the item's unique id, and context is input id, labels, schema_id, and model name
+            final_results_to_be_stored_in_alegre = {'documents': [
+                {'doc_id': str(uuid.uuid4()),  # adding a unique id for each item to not rely on the input id for uniqueness
+                 'content': items[i]['text'],
```

Comment:
Annoyingly, this parameter needs to be `text`, I believe.

Reply:
At least for /text/bulk_similarity/ we replace `text` with `content`:

```
# Rename "text" to "content" if present
if 'text' in params:
  params['content'] = params.get('text')
  del params["text"]
```

We can use both `content` and `text` for bulk but it ultimately gets renamed to `content` in Alegre.